ui/pages/DataManagement/reducers.js

```diff
@@ -75,11 +76,27 @@ export const uploadRnaSeq = values => (dispatch) => {
   })
 }
 
+export const uploadPhenoPri = values => (dispatch) => {
+  let successResponseJson = null
+  return new HttpRequestHelper(
+    '/api/data_management/load_phenotype_pri_data',
+    (responseJson) => {
+      successResponseJson = responseJson
+    },
+    (e) => {
+      successResponseJson = { warnings: [e.message] }
```

Comment:
There is no reason for you to write your own error handling here, it would be better if this behaved similar to how `uploadQcPipelineOutput` works